app/javascript/mastodon/features/account/components/account_note.js

```diff
@@ -140,7 +170,11 @@ class AccountNote extends ImmutablePureComponent {
 
   render () {
     const { account, intl } = this.props;
-    const { value, saved } = this.state;
+    const { value, saved, editable } = this.state;
+    const classNames = classnames('account__header__account-note__display', {
+      'empty': !value,
+    });
+    let emojifiedValue = emojify(escapeTextContentForBrowser(value), []).replace(/\r?\n/g, '<br />');
```

Comment:
If you style the content with `white-space: pre-wrap;`, the replacing at the end should not be needed.